charts/node/templates/statefulset.yaml

```diff
@@ -548,7 +548,9 @@ spec:
                 It is not a security risk unless they are exposed publicly.
                 */}}
                 --unsafe-rpc-external \
```

Comment:
There will be a problem because we always setting `--unsafe-rpc-external`, it was not a problem when this was only affecting the "internal" http port but we can't set it on the ws port for RPC nodes.